treeherder/etl/bugzilla.py

```diff
@@ -16,7 +16,7 @@ def fetch_intermittent_bugs(offset, limit):
         'keywords': 'intermittent-failure',
         'chfieldfrom': '-1y',
         'include_fields': ('id,summary,status,resolution,op_sys,cf_crash_signature,'
-                           'keywords,last_change_time'),
+                           'keywords,last_change_time, whiteboard, creation_time'),
```

Comment:
I think the references to `creation_time` here and in the test fixtures are now unused?